metabonding/src/util.rs

```diff
@@ -0,0 +1,61 @@
+elrond_wasm::imports!();
```

Comment:
I really don't like the usage of `util` for encapsulating a lot of unrelated functionalities. This file is 3 or 4 different modules in one. Even if for some we will have one function/module I still think it's easier to maintain (and less prone to future refactors) if we have: `access_control` for `require_caller_owner_or_signer`, crypto for `verify_signature`, `math` for `range` and `ratio`, `storage` for storage stuff